nodes/cma/nodes-cma-rn/nodes-cma-autoscaling-custom-rn.adoc

```diff
@@ -26,49 +26,52 @@ The following table defines the Custom Metrics Autoscaler Operator versions for
 
-|2.13.1
+|2.14.1
 |4.13
 |General availability
 
-|2.13.1
+|2.14.1
 |4.12
 |General availability
 |===
 
-[id="nodes-pods-autoscaling-custom-rn-2131_{context}"]
-== Custom Metrics Autoscaler Operator 2.13.1 release notes
+[id="nodes-pods-autoscaling-custom-rn-2141_{context}"]
+== Custom Metrics Autoscaler Operator 2.14.1 release notes
 
-This release of the Custom Metrics Autoscaler Operator 2.13.1-421 provides a new feature and a bug fix for running the Operator in an {product-title} cluster. The following advisory is available for the link:https://access.redhat.com/errata/RHBA-2024:4837[RHBA-2024:4837].
+This release of the Custom Metrics Autoscaler Operator 2.14.1-454 provides a CVE, a new feature, and bug fixes for running the Operator in an {product-title} cluster. The following advisory is available for the link:https://access.redhat.com/errata/RHBA-TBD[RHBA-TBD].
 
 [IMPORTANT]
 ====
 Before installing this version of the Custom Metrics Autoscaler Operator, remove any previously installed Technology Preview versions or the community-supported version of Kubernetes-based Event Driven Autoscaler (KEDA).
 ====
 
-[id="nodes-pods-autoscaling-custom-rn-2131-new_{context}"]
+[id="nodes-pods-autoscaling-custom-rn-2141-new_{context}"]
 === New features and enhancements
 
-[id="nodes-pods-autoscaling-custom-rn-2131-new-ca_{context}"]
-==== Support for custom certificates with the Custom Metrics Autoscaler Operator 
+[id="nodes-pods-autoscaling-custom-rn-2141-new-ca_{context}"]
+==== Support for the cron trigger with the Custom Metrics Autoscaler Operator
 
-The Custom Metrics Autoscaler Operator can now use custom service CA certificates to connect securely to TLS-enabled metrics sources, such as an external Kafka cluster or an external Prometheus service. By default, the Operator uses automatically-generated service certificates to connect to on-cluster services only. There is a new field in the `KedaController` object that allows you to load custom server CA certificates for connecting to external services by using config maps. 
+The Custom Metrics Autoscaler Operator can now use the Cron trigger to scale pods based on an hourly schedule. When your specified time frame starts, the Custom Metrics Autoscaler Operator scales pods to your desired amount. When the time frame ends, the Operator scales back down to the previous level.
 
-For more information, see xref:../../../nodes/cma/nodes-cma-autoscaling-custom.adoc#nodes-cma-autoscaling-custom-ca_nodes-cma-autoscaling-custom[Custom CA certificates for the Custom Metrics Autoscaler].
+For more information, see link:https://keda.sh/docs/2.14/scalers/cron/[Cron] in the Keda documentation.
+////
+If I can get this module reviewed on time for CMA 2.14.1 GA, add this link in place of the Keda docs link.
+For more information, see xref: nodes/cma/nodes-cma-autoscaling-custom-trigger.html#nodes-cma-autoscaling-custom-trigger-cron_nodes-cma-autoscaling-custom-trigger[Understanding the cron trigger].
```

Comment:
Don't forget to add ../../../ or two of them, however many directories we're scaling. This isn't an error, just a reminder.